mixer/pkg/protobuf/yaml/dynamic/valueTypeEncoder.go

```diff
@@ -19,6 +19,10 @@ import (
 
 	"github.com/gogo/protobuf/protoc-gen-gogo/descriptor"
 
+	"time"
```

Comment:
nit: imports should be in three blocks: standard library, third party, first party. Move time up to the top block and gogo proto types up beside the gogo descriptor import

Reply:
done.